src/tests/ftest/launch.py

```diff
@@ -345,15 +345,17 @@ def run_command(cmd):
 
     """
     print("Running {}".format(" ".join(cmd)))
-    process = subprocess.Popen(
-        cmd, stdout=subprocess.PIPE, stderr=subprocess.STDOUT,
-        universal_newlines=True)
-    stdout, _ = process.communicate()
-    retcode = process.poll()
+    try:
+        process = subprocess.Popen(
```

Comment:
(pylint-consider-using-with) Consider using 'with' for resource-allocating operations